pruner/light/pruner.go

```diff
@@ -3,15 +3,30 @@ package light
 import (
 	"context"
 
+	"github.com/ipfs/boxo/blockservice"
+
 	"github.com/celestiaorg/celestia-node/header"
+	"github.com/celestiaorg/celestia-node/share/ipld"
 )
 
-type Pruner struct{}
+type Pruner struct {
+	bserv blockservice.BlockService
```

Comment:
Perhaps you can use blockstore as input and wrap it with offline exchange. It will ensure block service doesn’t try to request data from network to delete it right after.

Reply:
It is fixed. Check the contructor. Also Vlad resolved himself